devices/org.osbuild.lvm2.lv

```diff
@@ -44,11 +44,20 @@ SCHEMA = """
   "volume": {
     "type": "string",
     "description": "Logical volume to active"
-  }
+  },
```

Comment:
???

Reply:
Thanks and sorry, must have happend during the conflict fix.